test/dynamo/test_exceptions.py

```diff
@@ -545,6 +545,34 @@ def fn(x, d, key):
         self.assertEqual(ref[0], res[0])
         self.assertEqual(ref[1], res[1])
 
+    @make_dynamo_test
+    def test_raise_from_None_2(self):
+        def fn():
+            try:
+                raise ValueError
+            except Exception:
+                raise TypeError from None
```

Comment:
Do we support exceptions with messages? e.g. TypeError("foo")

Reply:
Yep.